region/tourian/main/Blue Hopper Room.json

```diff
@@ -472,11 +475,14 @@
       },
       "requires": [
         "canShinechargeMovementTricky",
```

Comment:
Thinking this should be fine? It might require a few more shinecharge frames though. There was a [2, 1] requiring 160 frames, so I left it off there.